internal/cmd/schema-registry/command_schema_create.go

```diff
@@ -107,19 +91,26 @@ func (c *command) schemaCreate(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	request := srsdk.RegisterSchemaRequest{
-		Schema:     string(schema),
+	normalize, err := cmd.Flags().GetBool("normalize")
+	if err != nil {
+		return err
+	}
+
+	cfg := &RegisterSchemaConfigs{
+		Subject:    subject,
+		SchemaPath: &schema,
```

Comment:
Can this be changed to just a string, instead of a pointer to string? As far as I can tell, `SchemaPath` only looks like it's used in one place as an argument to `os.ReadFile`, which takes a string anyway.